src/features/linkApplicationToCustomer/utils.ts

```diff
@@ -19,15 +19,32 @@ export const getFilteredCustomersData = (data?: FILTERED_CUSTOMERS): CustomerDat
   if (!data?.profiles) return [];
 
   return data.profiles.edges.reduce<CustomerData[]>((acc, edge) => {
-    const { id, firstName, lastName, primaryAddress, berthLeases, customerGroup } = (edge as PROFILE_EDGE)
-      .node as PROFILE_NODE;
+    const {
+      id,
+      firstName,
+      lastName,
+      primaryAddress,
+      berthLeases,
+      customerGroup,
+      winterStorageLeases,
+    } = (edge as PROFILE_EDGE).node as PROFILE_NODE;
 
     const berths = berthLeases?.edges
       .map((edge) => edge?.node?.berth?.pier.properties?.harbor.properties?.name)
       .join(', ');
 
-    // TODO resolve place names
-    const winterStoragePlaces = '';
+    const winterStoragePlaces = winterStorageLeases?.edges
```

Comment:
Is this unit tested somewhere?

Reply:
I added mock data for this, but I could break it off into a separate function and test it more robustly.